script/run-isolated-tests.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# Get the list of subdirectories that contain apps and
+# extract substrings from the paths for the following regex.
+APP_SUBPATHS="$(find src -name manifest.json | sed -E 's/\/manifest.json//g' | xargs | sed 's/ /|/g')"
+
+# Match changed files to the app paths determined above
+FILES_CHANGED="$(git diff --name-only master)"
+APP_SUBPATHS_CHANGED="$(echo $FILES_CHANGED | grep -oE "($APP_SUBPATHS)")"
+NUM_APPS_CHANGED=$(echo "$APP_SUBPATHS_CHANGED" | wc -l)
+
+# Run tests only within the apps that have changed.
```

Comment:
Is this something needed specifically for CircleCI?

Reply:
Nope, not specific. This is just something I'm trying out as a potential general build process improvement.

Reply:
If we want to hold off on this and keep the workflow as similar to Jenkins as possible, I can just have the unit testing job be a `yarn test:coverage` without parsing diffs.

Reply:
I approved it as is since Circle is the only thing running this, but I'd also be ok with saving the test isolation for another PR and just sticking with `yarn test:coverage` for now